xl_auth/templates/users/inspect.html

```diff
@@ -0,0 +1,169 @@
+            <dd>{{ permissions_modified }}</dd>
+
+            <dt>{{ _('Collections Created') }}:</dt>
+            <dd>{{ collections_created }}</dd>
+
+            <dt>{{ _('Collections Modified') }}:</dt>
+            <dd>{{ collections_modified }}</dd>
+
+            <dt>{{ _('Clients Created') }}:</dt>
+            <dd>{{ clients_created }}</dd>
+
+            <dt>{{ _('Clients Modified') }}:</dt>
+            <dd>{{ clients_modified }}</dd>
+
+            <dt>{{ _('Last Modified') }}:</dt>
+            <dd>
+                {{ user.modified_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.inspect', user_id=user.modified_by.id) }}">
+                    {{ user.created_by.full_name }}
+                </a>
+            </dd>
+
+            <dt>{{ _('Created At') }}:</dt>
+            <dd>
+                {{ user.created_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.inspect', user_id=user.created_by.id) }}">
+                    {{ user.created_by.full_name }}
+                </a>
+            </dd>
+        </dl>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>{{ _('Permissions') }}</strong>
+        </div>
+        <table class="table table-hover">
+            <thead>
+            <tr>
+                <th class="col-md-1">{{ _('Collection') }}</th>
+                <th class="col-md-2 word-break-all">{{ _('Registrant') }}</th>
+                <th class="col-md-2 word-break-all">{{ _('Cataloger') }}</th>
+                <th class="col-md-2 word-break-all">{{ _('Cataloguing Administrator') }}</th>
+                <th class="col-md-2">{{ _('Active Collection') }}</th>
+                <th class="col-md-3">{{ _('Last Modified') }}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for permission in user.permissions | sort(attribute='collection.code') %}
+                <tr>
+                    <td>{{ permission.collection.code }}</td>
+                    <td class="{{ "bool-value-{}".format(permission.registrant).lower() }}">
+                        {{ _('Yes') if permission.registrant else _('No') }}
+                    </td>
+                    <td class="{{ "bool-value-{}".format(permission.cataloger).lower() }}">
+                        {{ _('Yes') if permission.cataloger else _('No') }}
+                    </td>
+                    <td class="{{ "bool-value-{}".format(permission.cataloging_admin).lower() }}">
+                        {{ _('Yes') if permission.cataloging_admin else _('No') }}
+                    </td>
+                    <td class="{{ "bool-value-{}".format(permission.collection.is_active).lower() }}">
+                        {{ _('Yes') if permission.collection.is_active else _('No') }}
+                    </td>
+                    <td>{{ permission.modified_at.strftime('%y-%m-%d') }}, {{  _('by') }}
```

Comment:
Do we need this info here? Since this is for looking at a use, I feel we're probably not *that* interested in when a permission was modified?

Reply:
Modified will generally be identical to created. So I think it's a pretty straightforward way of detecting if something fishy has happened to a user.. This is how it looks like in reality:

![screen shot 2017-11-27 at 11 05 01 am](https://user-images.githubusercontent.com/786326/33261366-deafad0e-d362-11e7-9ed1-c01629ffffae.png)